### DISCUSSION TITLE
高级账户相关说明（包括：绑定 · 授权 · 找回 UID · 被锁定的处理等）

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#1577] [高级账户]
# 重要提示

> 嗨，简悦全新的 [教程中心](https://kb.simpread.pro/) 已上线，如果你是新用户想要配置保存到本地或快照等相关功能的话，请访问教程中心，如果只是查看高级账户相关功能，请使用下面的链接。

***

> 作为新用户的话，只需要了解下面几个教程。

- [绑定高级账户](https://github.com/Kenshin/simpread/discussions/1573)

- [在新设备上登录](https://github.com/Kenshin/simpread/discussions/1574)

- [使用授权登录](https://github.com/Kenshin/simpread/discussions/1570)

- [取消授权](https://github.com/Kenshin/simpread/discussions/1575)

- [重新授权](https://github.com/Kenshin/simpread/discussions/1576)

- [找回丢失的 UID （高级账户资格）](https://github.com/Kenshin/simpread/discussions/1571)

- [找回 UID 后如何恢复高级账户资格](https://github.com/Kenshin/simpread/discussions/1958)

- [重新找到 License（兑换码）](https://github.com/Kenshin/simpread/discussions/2756)

- [安装和配置同步助手](https://github.com/Kenshin/simpread/discussions/2754)

***

> 这是关于高级账户的全部相关汇总

#### 简悦的账户系统

[简悦的高级账户系统是什么](https://github.com/Kenshin/simpread/discussions/1959) 这篇文章会对你完全掌握简悦的高级账户有非常重要的帮助。

### 注册/登录/授权相关

- [首次绑定（购买后如何绑定高级账户）](https://github.com/Kenshin/simpread/discussions/1573)

- [重新授权（当前设备为高级账户）](https://github.com/Kenshin/simpread/discussions/1576)

- [取消授权（取消当前设备的高级账户资格）](https://github.com/Kenshin/simpread/discussions/1575)

- [登录（在新设备如何使用简悦的高级账户）](https://github.com/Kenshin/simpread/discussions/1574)

- [授权登录（如何使用第三方登录简悦的高级账户）](https://github.com/Kenshin/simpread/discussions/1570)
### 异常情况处理

- [如何找回丢失的 UID （无法恢复高级账户资格）](https://github.com/Kenshin/simpread/discussions/1571)

- [找回 UID 后如何恢复高级账户资格](https://github.com/Kenshin/simpread/discussions/1958)

- [无法激活高级会员的解决办法（提示服务器错误）](https://github.com/Kenshin/simpread/discussions/1567)

- [账户被锁定（或显示非法使用被锁定）的处理方法](https://github.com/Kenshin/simpread/discussions/1569)

### 其它

- [已购必读](http://ksria.com/simpread/docs/#/已购必读)

- [[教程\]多设备同步设置方案 #2991](https://github.com/Kenshin/simpread/discussions/2991)

- [免密码登录 #2974](https://github.com/Kenshin/simpread/discussions/2974)

**Comment by Kenshin** [comment]
# 使用腾讯问卷方式购买的疑问

当你使用支付宝购买并提交到 [腾讯问卷](https://wj.qq.com/s2/6918802/ffc8/) 后不会实时发送兑换码，需要 24 ~ 48 小时内发送，

![image](https://user-images.githubusercontent.com/81074/168743463-9512b0ef-2725-4d0d-a270-15f97c8caf58.png)

**Comment by ziyouzhe-studio** [comment]
高级用户如何查看已登陆了几台设备？

**Reply by Kenshin** [reply]
https://simpread.pro/account

**Comment by mahuft** [comment]
<img width="1720" alt="iShot_2024-07-19_10 35 31" src="https://github.com/user-attachments/assets/6d2eddcd-00f2-444d-a597-cd3cfc6d0a73">

【兑换码申领平台】还是没有输入框

**Reply by Kenshin** [reply]
嗨，应该可以了，刚才在更新 CDN

你可以试试在隐私模式下进入这个网页，以免缓存造成的问题。

**Comment by skinnyshy** [comment]
腾讯问卷星提交失败
![image](https://github.com/user-attachments/assets/d494afb5-8ee5-4351-8b45-15fe280c8a36)
另外爱发电域名也换了 afdian.net打不开

**Reply by skinnyshy** [reply]
> 腾讯问卷星提交失败 ![image](https://private-user-images.githubusercontent.com/12553514/353773932-d494afb5-8ee5-4351-8b45-15fe280c8a36.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI0MTc5MjksIm5iZiI6MTcyMjQxNzYyOSwicGF0aCI6Ii8xMjU1MzUxNC8zNTM3NzM5MzItZDQ5NGFmYjUtOGVlNS00MzUxLThiNDUtMTVmZTI4MGM4YTM2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzMxVDA5MjAyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTY1YjY3NjQ3YTJjMTViODU3NWEwNDc3M2FiNTEyYWY5OWMxZWE2YjdiY2JhMmY2NjIzODgyNWQxZjhlNGMyNWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.I_jNR39gFZ_mIXlvyF3iS27pm7q2ozlrg7D6IiTNIP0) 另外爱发电域名也换了 afdian.net打不开

找到原因了，是Adguard规则的原因

**Reply by Kenshin** [reply]
`afdian.net` 已经暂时改为 https://afdian.com 简悦官方 http://simpread.pro/price 已经更新。

所以不知道你是在哪里得到的 `afdian.net` 方便的话请告知我可以更新它。

**Comment by jiang-zx** [comment]
最多支持授权三台设备，目前发现同一个电脑，不同的浏览器，也会限制。
所以三台设备是不是有点少了？

**Reply by Kenshin** [reply]
> 目前发现同一个电脑，不同的浏览器，也会限制。

同一台设备，不限浏览器（也就是说，这种情况算一台设备）

但个别情况下：多显示器的话，会被误判（**仅为个例**）
